tensorrt_llm/llmapi/llm_utils.py

```diff
@@ -806,6 +819,17 @@ def build_task(engine_dir: Path):
 
         return self.get_engine_dir()
 
+    @print_traceback_on_error
+    @staticmethod
+    def _node_download_hf_model(
```

Comment:
Is it possible to add a multi-node task with HF model ID as the model? We may need a QA test at least. I am not sure this feature can always work across versions without a test.